src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -325,6 +153,104 @@ class ExternalIssueActions extends AsyncComponent {
   }
 }
 
+export class SentryAppExternalIssueActions extends React.Component {
+  static propTypes = {
+    group: PropTypes.object.isRequired,
+    sentryAppComponent: PropTypes.object.isRequired,
+    externalIssue: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      action: 'create',
+      showModal: false,
+    };
+  }
+
+  openCreate = () => {
+    this.setState({action: 'create'});
+  }
+
+  openLink = () => {
+    this.setState({action: 'link'});
+  }
+
+  closeModal = () => {
+    this.setState({showModal: false});
+  }
+
+  onClick = () => {
+    if (!this.props.externalIssue) {
+      this.setState({showModal: true});
+    }
+  }
+
+  get link() {
+    const {sentryAppComponent, externalIssue} = this.props;
+
+    let url = '#';
+    let displayName = `Link ${sentryAppComponent.sentryApp.name} Issue`;
+
+    if (externalIssue) {
+      url = externalIssue.url;
+      displayName = externalIssue.displayName;
+    }
+
+    return (
+      <React.Fragment>
+        <IntegrationIcon src={`icon-github`} />
```

Comment:
Is this going to change later?

Reply:
Heh, yea, it is. The component hard fails if it can't find the icon, so I just put something in here temporarily.

Reply:
You can use `src="icon-generic-box"` instead